src/main/java/com/airlines/model/Airplane.java

```diff
@@ -0,0 +1,81 @@
+package com.airlines.model;
+
+import java.time.LocalDate;
+
+public class Airplane {
+    private int id;
+    private String codeName;
+    private String model;
+    private LocalDate manufactureDate;
+    private int capacity;
+    private double flightRange;
+    private int crewId;
+
+    public Airplane() {
```

Comment:
It's always good idea to hid constructor and make entity immutable, when we could afford this.

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1269dee2e4effc40beaddfca6d85b7799518548f